_posts/2021-06-10-How-to-notify-team-members-immediately-when-you-get-a-response-for-one-of-your-leads.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2021-06-10
+title: How to notify team members immediately, when you get a response for one of your leads
+video_id: JC2yJnBXzLM
+description: How to add team members so that they canalsoreply to leads response.
+type:  
+sidebar:
+  - {id: step-1, text: Go to assistant tab}
+  - {id: step-2, text: go to notification}
+  - {id: step-3, text: Add email adress of members}
+  
+categories:
+  - How-To
+# set: getting-started
+# set_order: 3
+---
+
+Responding to leads is one of the key factors for Converting them into potential customers.
+
+Sometimes one assistant forgets to respond to leads.
+
+With 7Targets you can make this process more feasible by adding team members who can respond to leads assigned to that particular assistant.
```

Comment:
With 7Targets, you can make sure that leads responses do not fall through the cracks by adding other sales team members to receive the response notification from the assistant.